src/components/list/listItem/subitem/ExpandableTextArea.tsx

```diff
@@ -19,24 +21,36 @@ const defaultTruncateCount = 430;
 const ExpandableTextArea: React.FC<ExpandableTextAreaProps> = ({
   text,
   isClickable = false,
+  isEnrichHtml = false,
   onClick = () => {},
   truncateCount = defaultTruncateCount,
   showMoreStr = "Show More",
 }) => {
   const decodedText = decodeHtmlEntities(text);
-  const truncatedText = truncateText(decodedText, truncateCount);
   const doesNeedTruncation = decodedText.length > truncateCount;
 
   const [isExpanded, setIsExpanded] = useState(false);
 
   const onButtonClick = useCallback(() => {
     setIsExpanded((prev) => !prev);
   }, []);
+
+  // In the typography we cannot use white-space="pre" in sx because
+  // we need to format the http link to html
   return (
     <TextAreaBaseGrid>
       <Grid item md={12}>
         <Typography
-          variant="detailContent"
+          component="div"
+          dangerouslySetInnerHTML={{
```

Comment:
Could you try not to use `dangerouslySetInnerHTML`? Because it exposes our portal to XSS attacks.

Reply:
I disable the dangerouslySetInnerHTML for now, but we lost the ability to click on the link within the abstract